docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_sbir.scss

```diff
@@ -0,0 +1,82 @@
+@import "./variables";
+@import "./mixins";
+@import "./environments/percussion";
+@import "./breakpoints";
+
```

Comment:
It feels like moving to a BEM naming convention would help organization and future maintenance here